### DISCUSSION TITLE
best practice for corporate ca

### DISCUSSION THREAD

**Discussion by poblin-orange** [kairos-io/kairos#1284] [Q&A]
Hello
In my environement kairos k3s needs to pull images from a corporate docker registry, exposed on a corporate ca https url.
What is the best practices to achieve this ?
thks

**Comment by mauromorales** [comment]
I think this is still in progress, you can follow up https://github.com/kairos-io/kairos/issues/513 and https://github.com/kairos-io/kairos/issues/1288

**Reply by poblin-orange** [reply]
Not sure #513, looks different from my issue (which is runtime related, having the running k3s to be able to join runtime corporate registry).

I did achieve it with k3s registries.yaml configuration. Was wondering if ca could be set at os level.

**Comment by jimmykarily** [comment]
@poblin-orange you can install it here: https://github.com/kairos-io/packages/blob/974753f40809e341081912487ad6a7d03502a111/packages/static/kairos-overlay-files/files/system/oem/00_rootfs.yaml#L25 . This directory is writable and persistent.
